pkgs/by-name/sp/spotube/package.nix

```diff
@@ -1,63 +1,66 @@
 {
-  lib,
-  stdenv,
-  fetchurl,
-
   autoPatchelfHook,
   dpkg,
-  makeBinaryWrapper,
-  makeWrapper,
-  undmg,
-  wrapGAppsHook3,
-
+  fetchurl,
   glib-networking,
   gtk3,
+  lib,
   libappindicator,
   libnotify,
   libsoup_3,
+  makeBinaryWrapper,
+  makeWrapper,
   mpv-unwrapped,
-  xdg-user-dirs,
+  stdenv,
+  undmg,
   webkitgtk_4_1,
+  wrapGAppsHook3,
+  xdg-user-dirs,
 }:
```

Comment:
I don't personally like alphabetical ordering. My own policy is this: `lib`, `stdenv`, and group the rest into categories. Those categories are then sorted alphabetically.
Could you undo this change.

In general, when making PRs, I try to change as few things as possible, so it's easier to review.

Reply:
okay sorry.